core/src/main/java/org/apache/calcite/util/Util.java

```diff
@@ -481,22 +481,26 @@ public static void print(
    * prints <code>"x\"y"</code>.
    */
   public static void printJavaString(
-      PrintWriter pw,
+      Appendable pw,
       String s,
```

Comment:
The argument is not PrintWriter now, we should not name is as `pw` now

Reply:
True, I will rename it. Thanks!